app/assets/javascripts/discourse/app/controllers/preferences/second-factor.js

```diff
@@ -113,6 +114,31 @@ export default Controller.extend(CanCheckEmails, {
       .finally(() => this.set("resetPasswordLoading", false));
   },
 
+  disableAllModalTemplate() {
```

Comment:
Minor, but the `Template` in the name here is misleading. Maybe we can call this `disableAllMessage`?